File: src/Modules/Meetings/Infrastructure/Domain/Meetings/MeetingsDbContext.fs
module CompanyName.MyMeetings.Modules.Meetings.Infrastructure.Domain.Meetings.MeetingsDbContext

open System
open System.Data.SqlClient
open CompanyName.MyMeetings.Modules.Meetings.Domain.Meetings
open Microsoft.EntityFrameworkCore
open Dapper
open CompanyName.MyMeetings.Modules.Meetings.Infrastructure.Domain.Meetings.MeetingsDbModel

type MeetingsDbContext(connectionString: string) =
    inherit DbContext()
    
    [<DefaultValue>]
    val mutable meetingsDbSet: DbSet<DbMeeting>
    
    member public this.DbMeetings
        with get () = this.meetingsDbSet
        and set (v) = this.meetingsDbSet <- v
        
    
    override this.OnConfiguring (optionsBuilder: DbContextOptionsBuilder) =
        optionsBuilder.UseSqlServer(connectionString) |> ignore
    
    override this.OnModelCreating builder =
        builder.ApplyConfigurationsFromAssembly(this.GetType().Assembly) |> ignore
    

File: src/Modules/Meetings/Infrastructure/MeetingsDbContext.cs
using CompanyName.MyMeetings.Modules.Meetings.Infrastructure.Database;
using Microsoft.EntityFrameworkCore;

namespace CompanyName.MyMeetings.Modules.Meetings.Infrastructure
{
    public class MeetingsDbContext : DbContext
    {
        public DbSet<Member> Members { get; set; }
        
        public DbSet<MeetingGroupProposal> MeetingGroupProposals { get; set; }

        public MeetingsDbContext(DbContextOptions options) : base(options)
        {
        }
        
        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            modelBuilder.ApplyConfigurationsFromAssembly(this.GetType().Assembly);
        }

        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
        {
            // optionsBuilder.UseSqlServer("Server=localhost;Database=MyMeetings;User Id=sa;Password=SqlServer2019;");
        }
    }
}